LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs

```diff
@@ -0,0 +1,114 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// All Rights Reserved.
+
+using System;
+using HidSharp;
+
+namespace LibreHardwareMonitor.Hardware.Controller.AquaComputer;
+//TODO:
+//Implement set RGB Controls
```

Comment:
Do yous till want to do this?

Reply:
No. I've no RGB hardware. So I can't test it. But maybe someone else wants to add that later.

Reply:
OK,  please format this document to follow code style and I'll merge :)

Reply:
Thanks!